Sources/Commands/SwiftTestTool.swift

```diff
@@ -46,6 +47,8 @@ private enum Mode: Argument, Equatable, CustomStringConvertible {
         case "-s", "--specifier":
             guard let specifier = pop() else { throw OptionParserError.expectedAssociatedValue(argument) }
             self = .run(specifier)
+        case "--parallel":
+            self = .runInParallel
```

Comment:
Yeah there is no filtering currently, I think it should be a proposal or atleast discussed on swift-build-dev about the UI for the filtering. For eg we could support something like `--filter` of Lit where a partial match also works.